.github/workflows/test-runner-nightly-integration.yml

```diff
@@ -0,0 +1,91 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Test Runner Nightly Integration Tests
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+
+jobs:
+  test-runner-integration:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:12.3
+        env:
+          POSTGRES_PASSWORD: postgres
```

Comment:
Maybe add a comment that we only do it this way because it's local.

Reply:
Comments added to latest commit.